server/planning/events/events_template.py

```diff
@@ -82,6 +84,9 @@ class EventsTemplateResource(Resource):
         },
         "subject": {"type": "list", "schema": {"type": "dict"}, "readonly": True},
     }
+    # add embedded_planning in the templates data schema
+    _event_fields.setdefault("embedded_planning", events_schema["embedded_planning"])
```

Comment:
No need to use `setdefault` here, you can just place it underneath the `subject` entry above :point_up: